packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -1895,7 +1940,11 @@ import 'output-localization-file_en.dart' deferred as output-localization-file_e
           throwsA(isA<L10nException>().having(
             (L10nException e) => e.message,
             'message',
-            contains('Check to see if the plural message is in the proper ICU syntax format'),
+//             contains('''
```

Comment:
delete comment?

Reply:
Added a todo for the comment for upcoming work.